docs/usage/user-guide/data-source/index.md

```diff
@@ -0,0 +1,10 @@
+---
+id: data-source
+title: Creating a Data Source
```

Comment:
this page is not about creating a data source but about testing a data source, should be moved to another directory about tests